docs/events/api/timeline-template-api-update.asciidoc

```diff
@@ -0,0 +1,385 @@
+[[timeline-template-api-update]]
+=== Update Timeline or Timeline template
+
+Updates an existing Timeline or Timeline template.
+
+Use the `timeline` object's <<timeline-object-typeField, `timelineType`>> field
+to specify whether you're updating a Timeline or a Timeline template:
+
+* `"timelineType": "default"` updates an existing Timeline
+* `"timelineType": "template"` updates an existing Timeline template
```

Comment:
Adding end periods since grammatically these are complete sentences:
```suggestion
* `"timelineType": "default"` updates an existing Timeline.
* `"timelineType": "template"` updates an existing Timeline template.
```